engine/src/main/java/org/entando/entando/web/userpreferences/model/UserPreferencesRequest.java

```diff
@@ -99,6 +100,14 @@ public void setDefaultWidgetJoinGroups(List<String> defaultWidgetJoinGroups) {
         this.defaultWidgetJoinGroups = defaultWidgetJoinGroups;
     }
 
+    public Boolean getDisableContentMenu() {
```

Comment:
you should use lombok to reduce code duplication

Reply:
see previous comment